package.json

```
@@ -9,19 +9,17 @@
   "version": "3.13.0",
   "scripts": {
     "build": "npm run build:production",
-    "build-storybook": "cp .env.production .env && build-storybook -c .storybook -o .sb",
-    "build:acceptance": "cp .env.acceptance .env && TS_NODE_PROJECT=tsconfig.webpack.json webpack --config ./webpack.prod.ts",
-    "build:development": "TS_NODE_PROJECT=tsconfig.webpack.json webpack --config ./webpack.dev.ts",
-    "build:production": "cp .env.production .env && TS_NODE_PROJECT=tsconfig.webpack.json webpack --config ./webpack.prod.ts",
+    "build-storybook": "build-storybook -c .storybook -o .sb",
+    "build:acceptance": "TS_NODE_PROJECT=tsconfig.webpack.json webpack --config ./webpack.prod.ts",
+    "build:production": "TS_NODE_PROJECT=tsconfig.webpack.json webpack --config ./webpack.prod.ts",
     "clean": "npm run clean:dirs",
     "clean:dirs": "rimraf ./node_modules ./coverage ./coverage-legacy ./dist ./build",
     "deploy-storybook": "storybook-to-ghpages",
     "generate:sitemap": "node ./sitemap-generator/index.js",
     "lint": "npm run lint:js",
     "lint:js": "eslint --format=node_modules/eslint-formatter-pretty src modules test/e2e/cypress --ext js,jsx,ts,tsx",
-    "postinstall": "cp .env.development .env",
     "start": "TS_NODE_PROJECT=tsconfig.webpack.json webpack-dev-server --config webpack.dev.ts",
-    "start:prod": "cp .env.production .env && TS_NODE_PROJECT=tsconfig.webpack.json webpack-dev-server --config webpack.dev.ts",
+    "start:prod": "TS_NODE_PROJECT=tsconfig.webpack.json webpack-dev-server --config webpack.dev.ts",
```

Should this not be using `webpack.prod.ts`?

```suggestion
    "start:prod": "TS_NODE_PROJECT=tsconfig.webpack.json webpack-dev-server --config webpack.prod.ts",
```